app/views/shared/_navbar.html.slim

```diff
@@ -0,0 +1,28 @@
+nav.navbar.navbar-expand-lg.navbar-light
+  div.container
+    = link_to 'Cashy', root_path, class: 'navbar-brand'
+    button.navbar-toggler type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation"
+      span.navbar-toggler-icon
+
+    div#navbarSupportedContent.collapse.navbar-collapse
+      ul.navbar-nav.mr-auto
+        li.nav-item
+          a.nav-link href="" Something
```

Comment:
My proposal is to remove that nav-button.
It redundant at the moment and it looks a bit weird when I'm trying to click on it and nothing happens.
In a real commercial app, that thing mustn't come to the `master`. `master` should always be stable and just work.
Probably, I described it poorly, but let's never leave the broken code in the PR's.